src/Raven.Studio/typescript/components/pages/database/indexes/errors/IndexErrorsPanel.tsx

```diff
@@ -173,3 +169,22 @@ export function IndexErrorsPanel(props: IndexErrorsPanelProps) {
         </RichPanel>
     );
 }
+
+interface IndexErrorsPanelStatusProps {
+    isLoading: boolean;
+    hasErrors: boolean;
+    status: AsyncStateStatus;
+}
+function IndexErrorsPanelStatus({ isLoading, hasErrors, status }: IndexErrorsPanelStatusProps) {
+    return (
+        <>
+            {isLoading && (
+                <RichPanelStatus color="secondary" data-testid="loader">
+                    Loading
+                </RichPanelStatus>
+            )}
+            {!isLoading && hasErrors && <RichPanelStatus color="danger">Errors</RichPanelStatus>}
+            {!isLoading && !hasErrors && status === "success" && <RichPanelStatus color="success">OK</RichPanelStatus>}
+        </>
+    );
```

Comment:
Make early return to simplify conditions

```suggestion
    if (isLoading) {
        return (
            <RichPanelStatus color="secondary" data-testid="loader">
                Loading
            </RichPanelStatus>
        );
    }
    if (hasErrors) {
        return <RichPanelStatus color="danger">Errors</RichPanelStatus>;
    }
    if (status === "success") {
        return <RichPanelStatus color="success">OK</RichPanelStatus>;
    }

    return null;
```

Reply:
Actually, I forgot about this design pattern, resolved.